app/models/concerns/waste_exemptions_engine/can_have_registration_attributes.rb

```diff
@@ -14,23 +14,23 @@ def partnership?
     end
 
     def operator_address
-      find_address_by_type(TransientAddress.address_types[:operator])
+      find_address_by_type("operator")
     end
 
     def contact_address
-      find_address_by_type(TransientAddress.address_types[:contact])
+      find_address_by_type("contact")
     end
 
     def site_address
-      find_address_by_type(TransientAddress.address_types[:site])
+      find_address_by_type("site")
     end
 
     private
 
     def find_address_by_type(address_type)
-      return nil unless transient_addresses.present?
+      return nil unless addresses.present?
```

Comment:
Excited to see this in here as I was about to do the same thing for the reg/t_reg overview page! Woop 👏

Reply:
Yes, when I saw your PR for the aliased associations I new it would help tidy up that concern.